core/src/scene/styleMixer.cpp

```diff
@@ -201,9 +201,6 @@ void StyleMixer::mergeFieldTakingLast(std::string key, Node target, const std::v
 void StyleMixer::mergeMapFieldTakingLast(std::string key, Node target, const std::vector<Node>& sources) {
```

Comment:
Just seen we were copying the keys here, we should use `const std::string&` instead. Can we update that at the same time?